### DISCUSSION TITLE
Copilot in NVIM and other LSPs

### DISCUSSION THREAD

**Discussion by kukula** [community/community#28813] [Copilot Conversations]
Hey! Is there a way to use Copilot in NVIM alongside other LSPs, like `tsserver` or `sorbet`?
Because when I enable them Copilot stops to suggest.

**Comment by dklymenk** [comment]
It is unclear from your comment what completion engine you are using, but if you are using cmp, you can explore this plugin.
https://github.com/zbirenbaum/copilot-cmp

**Comment by kukula** [comment]
@dklymenk I was using `SirVer/ultisnips` and `github/copilot.vim`, but after your answer I've tried `zbirenbaum/copilot-cmp`. It looked better than my current setup, so I changed it :) 
I've added sources like this
```
  sources = cmp.config.sources({
    { name = 'copilot', group_index = 2 },
    { name = 'nvim_lsp', group_index = 2 },
    { name = 'path', group_index = 2 },
    { name = 'ultisnips', group_index = 2 },
  },
```
but still I only get suggestions from macros and LSP but not Copilot(I paid for subscription and checked if it can auth).

**Reply by dklymenk** [reply]
Have you installed https://github.com/zbirenbaum/copilot.lua too? You need to use that as a replacement for the original service in order for cmp to query copilot.

**Reply by kukula** [reply]
yes! I did that too. is there any commands I can run to test the set up?
